pkg/models/devops/devops.go

```diff
@@ -218,7 +221,7 @@ func (d devopsOperator) ListPipelineObj(projectName string, limit, offset int) (
 	}
 	data, err := d.ksInformers.Devops().V1alpha3().Pipelines().Lister().Pipelines(projectObj.Status.AdminNamespace).List(labels.Everything())
 	if err != nil {
-		return api.ListResult{}, nil
+		return api.ListResult{}, err
```

Comment:
How about this one? It probably returns a nil items result.

Reply:
This api.ListResult{} will not be used ，just for return value type check.